Makefile

```diff
@@ -69,6 +69,12 @@ $(OPENSHIFT_GOIMPORTS):
 imports: $(OPENSHIFT_GOIMPORTS)
 	$(OPENSHIFT_GOIMPORTS) -m github.com/kcp-dev/kcp
 
+ifdef COUNT
+count := $(COUNT)
+else
+count := 5
+endif
+
 .PHONY: test-e2e
 test-e2e: install
-	go test -race -count 5 ./test/e2e... $(WHAT)
+	go test -race -count $(count) ./test/e2e... $(WHAT)
```

Comment:
mixed with other PR/commit

Reply:
Yeah, I broke it out from this one - can rebase once it lands